python/pyspark/ml/wrapper.py

```diff
@@ -223,3 +223,20 @@ def _call_java(self, name, *args):
         sc = SparkContext._active_spark_context
         java_args = [_py2java(sc, arg) for arg in args]
         return _java2py(sc, m(*java_args))
+
+
+class JavaCallable(object):
```

Comment:
I tried that before and it got a little messy with the definition of `_java_obj`, but I think I have an idea so let me give it another shot.